api/consensus.go

```diff
@@ -19,6 +21,55 @@ type ConsensusGET struct {
 	Difficulty   types.Currency    `json:"difficulty"`
 }
 
+type ConsensusFileContract struct {
+	FileSize           uint64                         `json:"filesize"`
+	FileMerkleRoot     crypto.Hash                    `json:"filemerkleroot"`
+	WindowStart        types.BlockHeight              `json:"windowstart"`
+	WindowEnd          types.BlockHeight              `json:"windowend"`
+	Payout             types.Currency                 `json:"payout"`
+	ValidProofOutputs  map[string]types.SiacoinOutput `json:"validproofoutputs"`
+	MissedProofOutputs map[string]types.SiacoinOutput `json:"missedproofoutputs"`
```

Comment:
Ok. My very strong preference is that we just return a normal types.Block. If someone does end up needing the hashes to be computed for them, we can return all the ids (txids, block id, output ids, etc.) in a separate struct that mirrors the block, except it only has ids as fields. But if nobody is using it, we don't need it.